Engine/src/Engine/UserInterface.cpp

```diff
@@ -0,0 +1,101 @@
+#include "UserInterface.h"
+
+ImGuiIO* UserInterface::s_IO = nullptr;
+ImGuiStyle* UserInterface::s_style = nullptr;
+
+std::map<std::string, ImFont*> UserInterface::s_fonts;
+
+//define the ImGUI vec4s for each colour
+ImVec4 UserInterface::red = ImVec4(1.0f, 0.0f, 0.0f, 1.0f);
+ImVec4 UserInterface::green = ImVec4(0.0f, 1.0f, 0.0f, 1.0f);
+ImVec4 UserInterface::blue = ImVec4(0.0f, 0.0f, 1.0f, 1.0f);
+ImVec4 UserInterface::white = ImVec4(1.0f, 1.0f, 1.0f, 1.0f);
+
+//initializes a GLFWwindow that can be used with ImGUI
```

Comment:
This comment seems inaccurate, unless I'm misunderstanding. 
Wouldn't this function initialize the UI and attach it to the provided GLFWwindow?

Reply:
it does indeed initialize the UI and not the window

Reply:
Ok, gotcha. In that case, can you update that comment when you get a chance?

Reply:
@KevinAlexV updated the comment